components/widgets/resources/src/main/resources/templates/geneVariantMacros.vm

```diff
@@ -158,7 +158,7 @@ $doc.display($prop.getName(), 'view', $fakeObj )##
 #set($variantObjects = $doc.getObjects($variantClass.name, "gene", $geneObj.getProperty('gene').value))
 |(% class="row-count" %)$geneNumber##
 #foreach($prop in $geneObj.xWikiClass.properties)##
-|(% class="$prop.translatedPrettyName gene #if ($prop.name == 'gene')mandatory#end gene-$geneObjNumber"##
+|(% class="$prop.translatedPrettyName gene #if ($prop.name == 'gene')mandatory#if($options.mode != 'edit' && !$services.vocabularies.hgnc.getTerm($geneObj.getProperty($prop.name).value)) custom-entry#end#end gene-$geneObjNumber"##
```

Comment:
`$geneObj.getProperty($prop.name).value` can be written as `$geneObj.getValue($prop.name)`.